code/graphics/software/font.cpp

```diff
@@ -688,7 +688,8 @@ void gr_get_string_size(int *w1, int *h1, const char *text, int len)
 	float w = 0.0f;
 	float h = 0.0f;
 
-	FontManager::getCurrentFont()->getStringSize(text, static_cast<size_t>(len), -1, &w, &h);
+	if (len != 0)
```

Comment:
`getStringSize()` Should have its own safety for when it detects a length <= 0, whats happening there that requires this additional safety?

Reply:
It's not "required" per se, it's just a small optimization.